### PR TITLE
FDC3 Survey Links

### PR DESCRIPTION
**Pull request by grizzwolf**
Adding FDC3 Survey Links in Marquee and temp using FeatureCallout for Survey (Hid Use Cases Temporarily)

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -137,11 +140,20 @@ class Index extends React.Component {
 
     const FeatureCallout = () => (
       <div  className="featureShowcaseSection  paddingBottom" style={{textAlign: 'center'}}>
-        <h2>Use Cases</h2>
-        <MarkdownBlock>{`Document business [use cases](${docUrl('use-cases/overview')}) that drive FDC3 interoperability standards.`}</MarkdownBlock>
+        <h2>FDC3 Survey</h2>
```
Comment (rikoe) [inline website/pages/en/index.js]: Does this change get rid of the use cases banner? Not sure we want that... Ideally a second banner?

Comment (grizzwolf) [reply website/pages/en/index.js]: It's a temporary change to highlight the survey for the next week (right?). I just hid the Markdown block so we can put it back up as soon as survey is done. I was asked to get this up quickly to help with draw attention to the survey. I tried a couple of other ways at first, but kept having issues getting them to work.

Comment (donbasuno) [reply website/pages/en/index.js]: Use Cases are still available in the top toolbar so given timing here - I am ok with temporarily removing the use case banner.

Thanks,

Johan

**Comment by nkolba** [comment]
@grizzwolf @rikoe @donbasuno 
I created an alternate CTA layout here:  https://github.com/FDC3/FDC3/pull/136

**Comment by brooklynrob** [comment]
Here's how this looks on my laptop after this PR was approved.

image

**Pull request merged**